awscli/customizations/wizard/devcommands.py

```diff
@@ -1,75 +0,0 @@
-# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Instead of deleting this command, could we just adapt the `wizard-dev` command to accept both v1 and v2 wizard files? I probably should have adapted this command in the first place instead of make it support only v2 commands, but this command is useful if you want to load it from outside the built-in wizard directory for the CLI.